src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -75,6 +75,15 @@ export const useAddressAutocomplete = (
     [apiKey]
   )
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const throttledFetchSuggestions = useCallback(
+    throttle(fetchSuggestions, 2000, {
```

Comment:
nitpick: throttle delay should be 500

Reply:
non-blocking: Would it make sense to rename `fetchSuggestions` `_fetchSuggestions` to make clear it is private and should never be called? Or we could not even declare it, just pass it directly to throttle. def not blocking.

Reply:
a throttle delay of 500 would mean half a second delay between requests, is that okay?

Reply:
yep, i think we may want to make it smaller in time - Could you make it a constant at the top of this file?